distributed/recreate_exceptions.py

```diff
@@ -91,14 +91,25 @@ def get_futures_error(self, future):
         """
         Ask the scheduler details of the sub-task of the given failed future
 
+        When a future evaluates to a status of "error", i.e., an exception
+        was raised in a task within its graph, we an get information from
+        the scheduler. This function gets the details of the specific task
+        that raised the exception and led to the error, but does not fetch
+        data from the cluster or execute the function. To actually run the
+        function locally and be able to debug, see
+        ``client.recreate_error_locally``.
```

Comment:
If you add this in a `See Also` section then you'll get an actual link in the API docs (this is worth checking out if you haven't already)